test/client/components/header.js

```diff
@@ -0,0 +1,24 @@
+import test from 'tape'
+import React from 'react'
+import { shallow } from 'enzyme'
+import Header from '../../../src/client/components/header.js'
+
+test('Header includes NM Logo, title and menubtn', t => {
+  t.plan(3)
```

Comment:
you are calling `t.end()` when your tests are done. `t.plan` is not for this use case, mostly it is useful for async stuff